privly.safariextension/scripts/background_scripts/handle_context_menu.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @fileOverview This file handles the context menu item, "Privly New Message"
+ *
+ * Workflow of the script:
+ * 1. When the user right clicks on the host page, the context information
+ *    is sent from post_new_link.js. The validate event is fired on right
+ *    click and the context information is checked if the content is editable.
+ *    If the content is not editable, the Privly New Message context menu
+ *    item is not shown.
+ * 2. If the content is editable, the context information is passed to the
+ *    posting_process.js file to finally open the posting application.
+ */
+
+
+/**
+ * If the content is not editable, do not show the Privly New Message
+ * context menu item.
+ *
+ * @param evt the validate event that is fired
+ *
+ */
+function onValidate(evt) {
```

Comment:
These functions should be namespaced. They are all sitting in the global namespace of the background page. Either put them onto a global namespace, like `var Privly = {...};`, or into a file-specific namespace like `var contextMenu = {...};`